go/libraries/doltcore/sqle/database.go

```diff
@@ -824,7 +824,7 @@ func (db Database) removeTableFromAutoIncrementTracker(
 }
 
 // CreateTable creates a table with the name and schema given.
-func (db Database) CreateTable(ctx *sql.Context, tableName string, sch sql.PrimaryKeySchema) error {
+func (db Database) CreateTable(ctx *sql.Context, tableName string, sch sql.PrimaryKeySchema, collation sql.CollationID) error {
```

Comment:
is `collation sql.CollationID` not part of `sql.PrimaryKeySchema`?

Reply:
It is not. I think it may belong outside of the schema from the GMS standpoint, along with the implementation engine (of which we only support InnoDB). In Dolt though, we pass along everything in the schema, so it’s far more straightforward to store it there.